src/components/Review.jsx

```diff
@@ -1,17 +1,45 @@
-import { Comments } from "./Comments";
+import axios from "axios";
+import { useState } from "react";
 
 export function Review({ title, designer, owner, review_img_url, category, created_at, votes, review_body, review_id }) {
+  const [currentVotes, setCurrentVotes] = useState(votes);
+  const [isDisabled, setIsDisabled] = useState(false);
+
+  function handleVoting(e) {
+    const incVotes = e.target.classList.contains("UpVote") ? 1 : -1;
+    axios
+      .patch(`https://be-nc-games.onrender.com/api/reviews/${review_id}`, { inc_votes: incVotes })
+      .then((response) => {
+        setCurrentVotes(votes + incVotes);
+        setIsDisabled(true);
```

Comment:
I would like you to use optimistic rendering for your voting functionality. This provides even more immediate feedback and you can do the patch after you've setCurrentVotes. 

When you're optimistic rendering, if an error occurs what would you need to do?